rust-code-analysis-cli/src/main.rs

```diff
@@ -587,5 +565,3 @@ fn main() {
         }
     }
 }
-
-// cargo run --release -- -p ../mozilla-central.hg/ -j24 -X "**/third_party/rust/**/*.*" -X "**/*.mako.rs" -X "**/js/src/jit/**/*MacroAssembler*" -X "**/js/src/vm/Opcodes.h" -X "**/obj-x86_64-pc-linux-gnu/**/*.*" -i -c
```

Comment:
If you want to remove this comment, please submit a separate PR as it's not related to this change